src/Microsoft.AspNetCore.Mvc.Core/Internal/AuthorizationApplicationModelProvider.cs

```diff
@@ -61,18 +61,18 @@ public void OnProvidersExecuting(ApplicationModelProviderContext context)
             }
         }
 
-        private AuthorizeFilter GetFilter(IEnumerable<IAuthorizeData> authData)
+        public static AuthorizeFilter GetFilter(IAuthorizationPolicyProvider policyProvider, IEnumerable<IAuthorizeData> authData)
```

Comment:
Ideally this would be a static factory method on `AuthorizeFilter`, but this API should go away eventually, so lets keep it where it is.